pkg/chartutil/requirements.go

```diff
@@ -106,3 +116,138 @@ func LoadRequirementsLock(c *chart.Chart) (*RequirementsLock, error) {
 	r := &RequirementsLock{}
 	return r, yaml.Unmarshal(data, r)
 }
+
+// ProcessRequirementsConditions disables charts based on condition path value in values
+func ProcessRequirementsConditions(reqs *Requirements, cvals Values) {
+	var cond string
+	var conds []string
+
+	if reqs != nil && len(reqs.Dependencies) > 0 {
```

Comment:
One way to make this shallower might be to check if this is nil or len 0, and return. Then you could bump the for loop up a level. That might make it a tiny bit easier to read, since this gets pretty deeply nested.